build/restore.ps1

```diff
@@ -1,7 +1,14 @@
 $root = $PSScriptRoot
+$packages = "$root\..\packages"
+
 $nuget = & "$root\Get-Nuget.ps1"
 
-& $nuget restore $root\..\PortabilityTools.sln
+if (!(Test-Path $packages))
+{
+    New-Item $packages -ItemType Directory
```

Comment:
The recommendations I've seen for powershell scripts is to use built-in names, not aliases, in case they have been changed to something else.